types/node/ts4.0/base.d.ts

```diff
@@ -0,0 +1,19 @@
+// NOTE: These definitions support NodeJS and TypeScript 3.7.
+
+// NOTE: TypeScript version-specific augmentations can be found in the following paths:
+//          - ~/base.d.ts         - Shared definitions common to all TypeScript versions
+//          - ~/index.d.ts        - Definitions specific to TypeScript 2.1
+//          - ~/ts3.7/base.d.ts   - Definitions specific to TypeScript 3.7
+//          - ~/ts3.7/index.d.ts  - Definitions specific to TypeScript 3.7 with assert pulled in
```

Comment:
I'm a bit confused by these comments but I just copied what I saw in `types/node/ts3.6/base.d.ts` and updated the version from `3.5` to `3.7`.

Reply:
These comments are just stale and need to be updated to match the new file layout related to `"typesVersions"`.